Source/WebKit/Platform/spi/Cocoa/ExtensionKitSPI.h

```diff
@@ -27,8 +27,11 @@
 
 #if USE(EXTENSIONKIT)
 
-#if __has_include(<ServiceExtensions/ServiceExtensions_Private.h>)
-#import <ServiceExtensions/ServiceExtensions_Private.h>
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+#if __has_include(<ServiceExtensions/SEServiceManager_Private.h>)
+#import <ServiceExtensions/SEServiceManager_Private.h>
```

Comment:
Is this change because `SEServiceManager_Private.h` is not imported by `ServiceExtensions_Private.h`? If so, that's a mistake; `ServiceExtensions_Private.h` should import all the private headers (I know that's not something you can fix in this PR though).

Also, doesn't `ServiceExtensions_Private.h` import things we need that `SEServiceManager_Private.h` doesn't, like `SEProcess_Private.h`?

Reply:
Ah, this change was not intentional, I reverted this part.